### PR TITLE
[styles][refactor] -navbar responsiveness & minute fixes

### PR DESCRIPTION
**Pull request by abhij1607**
- added menu modal drawer
- refactored search input and search button  styles
- refactored styles & structure to make Navbar more responsive

### CODE REVIEW

File: src/styles/nav.css [hunk 1]
```diff
@@ -37,6 +32,106 @@
...
+.bar2,
+.bar3 {
+  width: 35px;
+  height: 5px;
+  background-color: var(--txt-clr-secondary);
+  margin: 6px 0;
+  transition: 0.4s;
+}
+.cross .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-9px, 6px);
```
Comment (GuptaEesh) [inline src/styles/nav.css]: I don't think this is needed.

**Review by GuptaEesh — COMMENTED** [review]
The code looks clean.
Good component hierarchy.
Good to merge

**Review by AnuragSahu11 — APPROVED** [review]
Good use of Css variables , proper class names used, Good to merge

File: src/styles/nav.css (hunk 1 above)
Comment (abhij1607) [reply src/styles/nav.css]: thanks for pointing it out, I'll remove it

**Pull request merged**